gradle/base.gradle

```diff
@@ -0,0 +1,18 @@
+android {
```

Comment:
Can you undo this change? I'd rather keep the build files self-contained. There aren't that many modules and I would like each one, especially the ones that get published, to be very explicit.

Reply:
I can revert them. 
But I think I just extracted common parts of modules' gradle, and properties could be overrided.

Reply:
Please revert them